specification/cognitiveservices/data-plane/Language/preview/2022-02-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2742 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Language Authoring CLU APIs",
+    "version": "2022-02-01-preview"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{Endpoint}/language/analyze-conversations",
+    "useSchemePrefix": false,
+    "parameters": [
+      {
+        "$ref": "#/parameters/Endpoint"
+      }
+    ]
+  },
+  "securityDefinitions": {
+    "apiKeyHeader": {
+      "type": "apiKey",
+      "name": "Ocp-Apim-Subscription-Key",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apiKeyHeader": []
+    }
+  ],
+  "paths": {
+    "/projects": {
+      "get": {
+        "description": "Returns the list of projects.",
+        "operationId": "Projects_GetProjectsList",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "top",
+            "description": "The number of projects to be returned.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "skip",
+            "description": "The offset in the response.",
+            "type": "string"
+          },
+          {
+            "in": "query",
+            "name": "maxpagesize",
+            "description": "The maximum number of projects per page.",
+            "type": "string"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The metadata of projects.",
+            "schema": {
+              "$ref": "#/definitions/ProjectGetObjectList"
+            }
+          },
+          "400": {
+            "description": "This error can be returned if the request's parameters are incorrect meaning the required parameters are missing, malformed, or too large.\\r\\n\\r\\nThis error can be returned if the request's body is incorrect meaning the JSON is missing, malformed, or too large."
```

Comment:
All error code needs an error schema. https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#handling-errors

Also it needs a "default" for other errors (e.g. 5xx).

And it need to have `x-ms-error-response`. https://azure.github.io/autorest/extensions/#x-ms-error-response